mix.exs

```diff
@@ -20,7 +20,7 @@ defmodule ErllambdaElixirExample.MixProject do
   defp deps do
     [
       {:erllambda, "~> 2.0"},
-      {:mix_erllambda, "~> 1.0"},
+      {:mix_erllambda, "~> 1.1"},
```

Comment:
just in case it's 1.0.0 now in hex and will be 1.1.0 if that matter @velimir

Reply:
it's ok, according to sevmer it should work.

Reply:
argh...can you push it to hex please @velimir, you should still have the right. 
smth is broke on my machine

Reply:
sure, I'll let you know when it's there 👍

Reply:
https://hex.pm/packages/mix_erllambda/1.1.0 is out